doc/api/os.md

```diff
@@ -1160,6 +1160,52 @@ The following error codes are specific to the Windows operating system:
   </tr>
 </table>
 
+### dlopen Constants
+
+If available on the operating system, the following signal constants
+are exported by `os.constants.dlopen`. See [man 3 dlopen][] for detailed
```

Comment:
@mscdex Can you point me at any examples of this? I can't find any, but [man 3 foo][] usage.